src/components/card-layout.ts

```diff
@@ -1,19 +1,39 @@
 import { LitElement, css, html } from "lit";
 import { customElement, property } from "lit/decorators.js";
+import { PAGE_STYLES } from "../const";
 
 @customElement("card-layout")
 export class CardLayout extends LitElement {
   @property() public header!: string;
+  @property() public casita!: string;
+  @property() public alignment: "center" | "left" = "left";
 
   render() {
+    let cardHeader = null;
+    if (this.header) {
+      cardHeader = html`
+        <h1 class="card-header">
+          ${this.casita
```

Comment:
This is better done with a slot. `card-layout` is a generic component and shouldn't have the notion of casitas.